libs/git_wrapper/Git_wrapper.ml

```diff
@@ -349,8 +349,20 @@ let git_log_json_format =
    \"contributor\": {\"commit_author_name\": \"%an\", \"commit_author_email\": \
    \"%ae\"}}"
 
+let get_month_ago_date () =
+  let yesterday = Common2.yesterday () in
+  let month_ago = Common2.month_before yesterday in
+  let after_date = Unix.gmtime month_ago in
+  let year = after_date.tm_year + 1900 in
+  let month = after_date.tm_mon + 1 in
+  let day = after_date.tm_mday in
+  Printf.sprintf "%04d-%02d-%02d" year month day
+
 let get_git_logs () : string list =
-  let cmd = Bos.Cmd.(v "git" % "log" % git_log_json_format) in
+  (* TODO Allow this to be configurable past 30 days? *)
```

Comment:
at this point would be better to make it an argument of get_git_logs
like get_git_logs ~since:(Common2.month_before (Common2.yesterday ()) 
so you specify this at the call site.
and maybe have an helper function here like  time_to_string_for_git_after (date : Common2.float_time) : string